modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectEntryLocalServiceImpl.java

```diff
@@ -5298,6 +5302,29 @@ private void _validateListTypeEntryKey(
 		}
 	}
 
+	private void _validateObjectEntryFolderId(
+			long groupId, long objectEntryFolderId)
+		throws PortalException {
+
+		if (objectEntryFolderId ==
+				ObjectEntryFolderConstants.
+					PARENT_OBJECT_ENTRY_FOLDER_ID_DEFAULT) {
+
+			return;
+		}
+
+		ObjectEntryFolder objectEntryFolder =
+			_objectEntryFolderService.getObjectEntryFolder(objectEntryFolderId);
```

Comment:
hi, I think this should be
```_objectEntryFolderLocalService```, as we should be calling Local services from Local services

Reply:
thanks!

Reply:
My question was not originating from easy or convenience, but about code consistency. Our expectations for this use case might differ from the object team's. We are only looking at it through CMS 2.0 while they would look at it with the whole framework in mind.

Reply:
At least I would not be happy if some other team would change things in our service Layer without our review.